aiida/backends/tests/tools/graph/test_age.py

```diff
@@ -0,0 +1,771 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+# pylint: disable=too-many-locals, too-many-statements
+"""AGE tests"""
+
+import numpy as np
+
+from aiida.tools.graph.age_entities import Basket, GroupNodeEdge
+from aiida.tools.graph.age_rules import (UpdateRule, ReplaceRule, RuleSequence, RuleSaveWalkers, RuleSetWalkers)
+
+from aiida.backends.testbase import AiidaTestCase
+from aiida.common.links import LinkType
+from aiida.orm import Group, Node, Data, CalculationNode, WorkflowNode, ProcessNode
```

Comment:
May I ask why? Because if the problem is only the long import lines, given that I have around a 100 uses of these `orm.<whatever>`, it doesn't sound like a good trade-off, right? Unless there are other arguments like code clarity (a loose `Node` could mean anything, whereas `orm.Node` is more specific and even easier to grep).